cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -7,7 +7,15 @@ class HttpCache
 
   # Language header and cookie are needed to separately cache language-specific pages.
   LANGUAGE_HEADER = %w(Accept-Language)
-  LANGUAGE_COOKIES = %w(language_ pm)
+
+  DEFAULT_COOKIES = [
+    # Language drop-down selection.
+    'language_',
```

Comment:
Note that this is different from the kind of behavior we saw with old language-mixing bugs, where a single cached page would get 'poisoned' by a mix of differently-localized renderings. That was caused by the language header/cookie not being present in the cache key, and we fixed this by actively removing headers/cookies and replacing them with a default (in our case `en-US` for the language header) whenever they're not present in the cache key.

With this PR the language header/cookie is always present in the cache key, so all pages will be served by separate language-specific caches.